src/filters/gobalException.filter.ts

```diff
@@ -21,7 +21,7 @@ export class GlobalExceptionFilter implements ExceptionFilter {
 
     let message: string | string[];
     let status: HttpStatus;
-
+    console.log(exception);
     switch (true) {
```

Comment:
its strange condition, I dont think its right

Reply:
then I misunderstand something, I could not approve `switch (true) {`

Reply:
done